code/game/objects/items/devices/motion_detector.dm

```diff
@@ -191,6 +191,13 @@
 	if(ishuman(A.loc))
 		return A.loc
 
+/obj/item/device/motiondetector/xm4
+
+/obj/item/device/motiondetector/xm4/get_user()
+	var/atom/A = loc
+	if(ishuman(A.loc))
+		return A.loc
```

Comment:
Doesn't this just do the same as the proc  on L183-185 we're casting to an /atom but not operating on it so I don't see why we need this override?